src/intl/index.js

```diff
@@ -59,12 +59,12 @@ function getMessageForLocale (id: MessageIdType, targetLocale: string): ?string
   return null
 }
 
-const messageFormatCache: {[MessageIdType]: IntlMessageFormat} = {}
+const messageFormatCache: {[MessageIdType]: typeof IntlMessageFormat} = {}
```

Comment:
New Flow setup reported an error here. `IntlMessageFormat` is a class imported from un-typed module.  When used directly it triggers: 
```
Cannot use `IntlMessageFormat` as a type because it is an `any`-typed value.
```